src/Template/Element/sentences/translation.ctp

```diff
@@ -1,44 +1,62 @@
 <?php
-use App\Lib\LanguagesLib;
+use Cake\Core\Configure;
 
-$showExtra = '';
-if ($isExtra) {
-    $showExtra = 'ng-if="vm.isExpanded"';
-}
-$translationUrl = $this->Url->build(array(
+$sentenceBaseUrl = $this->Url->build([
     'controller' => 'sentences',
     'action' => 'show',
-    $translation->id
-));
-$notReliable = $translation->correctness == -1;
+]);
+$audioBaseUrl = Configure::read('Recordings.url');
 ?>
-<div layout="row" layout-align="start center" <?= $showExtra ?>
-     class="translation <?= $notReliable ? 'not-reliable' : '' ?>">
+<div ng-repeat="translation in <?= $translations ?>" layout="row" layout-align="start center"
+     class="translation" ng-class="{'not-reliable' : translation.correctness === -1}">
+    
     <md-icon class="chevron">chevron_right</md-icon>
+
     <div class="lang">
-        <?= $this->Languages->icon(
-                $translation->lang,
-                [
-                    'width' => 30,
-                    'height' => 20
-                ]
-            ) ?>
+        <img class="language-icon" src="/img/flags/{{translation.lang ? translation.lang : 'unknown'}}.svg" />
```

Comment:
This should be changed to `<img ... ng-src="/img/flags ...` to avoid the 404 errors I've mentioned. See https://docs.angularjs.org/api/ng/directive/ngSrc